src/main/java/org/jenkins/plugins/statistics/gatherer/StatisticsConfiguration.java

```diff
@@ -220,7 +220,7 @@ public FormValidation doCheckBuildUrl(
             @QueryParameter("buildUrl") final String buildUrl) {
         if (buildUrl == null || buildUrl.isEmpty()) {
             return FormValidation.error("Provide valid Build URL. " +
-                    "For ex: \"http://ci.mycompany.com/api/builds\"");
+                    "For example: \"http://cistats.mycompany.com/api/build\"");
```

Comment:
This is just an example, why `cistats` is better than `ci`?

Reply:
To goal was to make it consistent. "cistats" is used in tests and particularly the help texts - see all the _src/main/resources/org/jenkins/plugins/statistics/gatherer/*/help-*.html_ files.